diff --git a/Cargo.toml b/Cargo.toml
index 541d14a..7ae892e 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -24,32 +24,37 @@ travis-ci = { repository = "rust-num/num" }
 optional = true # needs std
 #version = "0.2.0"
 #default-features = false
-git = "https://github.com/mesalock-linux/num-bigint-sgx"
+#git = "https://github.com/mesalock-linux/num-bigint-sgx"
+path = "../num-bigint-sgx"
 
 [dependencies.num-complex]
 #version = "0.2.0"
 #default-features = false
-git = "https://github.com/mesalock-linux/num-complex-sgx"
+#git = "https://github.com/mesalock-linux/num-complex-sgx"
+path = "../num-complex-sgx"
 
 [dependencies.num-integer]
 #version = "0.1.39"
 #default-features = false
-git = "https://github.com/mesalock-linux/num-integer-sgx"
+#git = "https://github.com/mesalock-linux/num-integer-sgx"
+path = "../num-integer-sgx"
 
 [dependencies.num-iter]
 #version = "0.1.37"
 #default-features = false
-git = "https://github.com/mesalock-linux/num-iter-sgx"
+path="../num-iter-sgx"
 
 [dependencies.num-rational]
 #version = "0.2.1"
 #default-features = false
-git = "https://github.com/mesalock-linux/num-rational-sgx"
+#git = "https://github.com/mesalock-linux/num-rational-sgx"
+path = "../num-rational-sgx"
 
 [dependencies.num-traits]
 #version = "0.2.5"
 #default-features = false
 git = "https://github.com/mesalock-linux/num-traits-sgx"
+branch="v1.1.2-backup"
 
 [dev-dependencies]
 
