diff --git a/Cargo.toml b/Cargo.toml
index 4026d70..fc21f97 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -38,12 +38,14 @@ sgx_tstd = { rev = "v1.1.2", git = "https://github.com/apache/teaclave-sgx-sdk.g
 [dependencies.num-integer]
 #version = "0.1.42"
 #default-features = false
-git = "https://github.com/mesalock-linux/num-integer-sgx"
+#git = "https://github.com/mesalock-linux/num-integer-sgx"
+path = "../num-integer-sgx"
 
 [dependencies.num-traits]
 #version = "0.2.11"
 #default-features = false
 git = "https://github.com/mesalock-linux/num-traits-sgx"
+branch = "v1.1.2-backup"
 
 #[dependencies.rand]
 #optional = true
@@ -58,18 +60,21 @@ version = "1.0"
 #default-features = false
 #features = ["std", "mesalock_sgx"]
 git = "https://github.com/mesalock-linux/serde-sgx"
+branch = "v1.1.2-backup"
 
 [dependencies.quickcheck]
 optional = true
 #version = "0.8"
 #default-features = false
 git = "https://github.com/mesalock-linux/quickcheck-sgx"
+branch = "v1.1.2-backup"
 
 [dependencies.quickcheck_macros]
 optional = true
 #version = "0.8"
 #default-features = false
 git = "https://github.com/mesalock-linux/quickcheck-sgx"
+branch = "v1.1.2-backup"
 
 #[dev-dependencies.serde_test]
 #version = "1.0"
