diff --git a/Cargo.toml b/Cargo.toml
index 1310986..3f24551 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -22,12 +22,14 @@ sgx_tstd = { rev = "v1.1.2", git = "https://github.com/apache/teaclave-sgx-sdk.g
 [dependencies.num-integer]
 #version = "0.1.38"
 #default-features = false
-git = "https://github.com/mesalock-linux/num-integer-sgx"
+#git = "https://github.com/mesalock-linux/num-integer-sgx"
+path = "../num-integer-sgx"
 
 [dependencies.num-traits]
 #version = "0.2.4"
 #default-features = false
 git = "https://github.com/mesalock-linux/num-traits-sgx"
+branch = "v1.1.2-backup"
 
 [features]
 default = ["std", "mesalock_sgx", "i128"]
