diff --git a/Cargo.toml b/Cargo.toml
index df9382756c..ab0a1a98b5 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -57,8 +57,8 @@ rand_pcg = { path = "rand_pcg", version = "0.2", optional = true }
 # Do not depend on 'getrandom_package' directly; use the 'getrandom' feature!
 # This is a dependency because: we forward wasm feature flags
 # This is renamed because: we need getrandom to depend on rand_core/getrandom
-getrandom_package = { git = "https://github.com/mesalock-linux/getrandom-sgx", package = "getrandom", optional = true }
-log = { git = "https://github.com/mesalock-linux/log-sgx", optional = true }
+getrandom_package = { git = "https://github.com/mesalock-linux/getrandom-sgx", package = "getrandom", optional = true, branch="v1.1.2-backup"}
+log = { git = "https://github.com/mesalock-linux/log-sgx", optional = true, branch="v1.1.2-backup"}
 sgx_tstd = { version = "1.0", rev = "v1.1.2", git = "https://github.com/apache/teaclave-sgx-sdk.git", optional = true }
 
 [dependencies.packed_simd]
diff --git a/rand_chacha/Cargo.toml b/rand_chacha/Cargo.toml
index b973a65039..b916789663 100644
--- a/rand_chacha/Cargo.toml
+++ b/rand_chacha/Cargo.toml
@@ -20,7 +20,7 @@ appveyor = { repository = "rust-random/rand" }
 
 [dependencies]
 rand_core = { path = "../rand_core", version = "0.5" }
-c2-chacha = { git = "https://github.com/mesalock-linux/cryptocorrosion-sgx" }
+c2-chacha = { path = "../../cryptocorrosion-sgx/stream-ciphers/chacha" }
 sgx_tstd = { rev = "v1.1.2", git = "https://github.com/apache/teaclave-sgx-sdk.git", optional = true }
 
 [features]
diff --git a/rand_core/Cargo.toml b/rand_core/Cargo.toml
index f3f8d64149..d7fc1555e7 100644
--- a/rand_core/Cargo.toml
+++ b/rand_core/Cargo.toml
@@ -25,8 +25,8 @@ serde1 = ["serde"] # enables serde for BlockRng wrapper
 mesalock_sgx = ["alloc", "sgx_tstd"]
 
 [dependencies]
-serde = { git = "https://github.com/mesalock-linux/serde-sgx", features = ["derive"], optional = true }
-getrandom = { git = "https://github.com/mesalock-linux/getrandom-sgx", optional = true }
+serde = { git = "https://github.com/mesalock-linux/serde-sgx", features = ["derive"], optional = true, branch="v1.1.2-backup"}
+getrandom = { git = "https://github.com/mesalock-linux/getrandom-sgx", optional = true, branch="v1.1.2-backup"}
 sgx_tstd = { rev = "v1.1.2", git = "https://github.com/apache/teaclave-sgx-sdk.git", optional = true }
 
 [package.metadata.docs.rs]
