diff --git a/Cargo.toml b/Cargo.toml
index 60eabe9..7d78086 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -108,25 +108,29 @@ pattern = []
 [dependencies.aho-corasick]
 #version = "0.7.6"
 optional = true
-git = "https://github.com/mesalock-linux/aho-corasick-sgx"
+#git = "https://github.com/mesalock-linux/aho-corasick-sgx"
+path = "../aho-corasick-sgx"
 
 # For skipping along search text quickly when a leading byte is known.
 [dependencies.memchr]
 #version = "2.2.1"
 optional = true
 git = "https://github.com/mesalock-linux/rust-memchr-sgx"
+branch = "v1.1.2-backup"
 
 # For managing regex caches quickly across multiple threads.
 [dependencies.thread_local]
 #version = "0.3.6"
 optional = true
 git = "https://github.com/mesalock-linux/thread_local-rs-sgx"
+branch = "v1.1.2-backup"
 
 # For parsing regular expressions.
 [dependencies.regex-syntax]
 path = "regex-syntax"
-version = "0.6.12"
+#version = "0.6.12"
 default-features = false
+branch = "v1.1.2-backup"
 
 [dependencies]
 sgx_tstd = { rev = "v1.1.2", git="https://github.com/apache/teaclave-sgx-sdk.git", optional = true }
