diff --git a/Cargo.toml b/Cargo.toml
index 1b2dd74..63ab906 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -10,8 +10,8 @@ Simple library for converting to and from base-58 strings.
 """
 
 [dependencies]
-num = { git = "https://github.com/mesalock-linux/num-sgx" }
 sgx_tstd = { rev = "v1.1.2", git = "https://github.com/apache/teaclave-sgx-sdk.git", optional = true }
+num = { path = "../num-sgx" }
 
 [dev-dependencies]
 #rand = "*"
