diff --git a/Cargo.toml b/Cargo.toml
index b36d554..fddd3c7 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -18,9 +18,9 @@ travis-ci = { repository = "serde-rs/json" }
 appveyor = { repository = "serde-rs/json" }
 
 [dependencies]
-serde = { git = "https://github.com/mesalock-linux/serde-sgx" }
-indexmap = { git = "https://github.com/mesalock-linux/indexmap-sgx", optional = true }
-itoa = { git = "https://github.com/mesalock-linux/itoa-sgx" }
+serde = { git = "https://github.com/mesalock-linux/serde-sgx", branch="v1.1.2-backup"}
+indexmap = { git = "https://github.com/mesalock-linux/indexmap-sgx", optional = true, branch="v1.1.2-backup"}
+itoa = { git = "https://github.com/mesalock-linux/itoa-sgx", branch = "v1.1.2-backup"}
 ryu = "1.0"
 sgx_tstd = { rev = "v1.1.2", git = "https://github.com/apache/teaclave-sgx-sdk.git", optional = true }
 
